### ISSUE TITLE
[Question] change default debug events timeframe in version 3.6.x

### ISSUE THREAD

**Issue by t0mZ4** [thingsboard/thingsboard#10035]
Is there a way to change the default settings here from Last 15 MIN to something else?

![image](https://github.com/thingsboard/thingsboard/assets/31729091/684a57f2-882b-47f1-9b92-dc9699c6f372)

**Comment by ashvayka** [comment]
Hi @t0mZ4 and @devaskim . We have intentionally changed the value to minimize DB load when a user opens this window. At the moment, it is hard-coded. To prioritize the issue, I need to understand why you would like to change this.

**Comment by devaskim** [comment]
I think the period can be configurable both through `thingsboard.yml` and tenant profile settings.

In our case a device may send timeseries once in a several hour or for some invertigration I need to get several packages at once during the some period, in majority of cases this period is more than 15 minutes.

**Comment by t0mZ4** [comment]
@ashvayka indeed, in 3.4.3 it was set to 1day, and we almost never had to change yet. In the 15min setting we have to change it multiple times during a debug/dev session.

So I can see how for some the 15min is suitable and performance oriented, but for us it becomes a tedious new step.

@devaskim where can we change this default 15min period?

**Comment by devaskim** [comment]
@t0mZ4 nowhere, I just suggested the way how it may be.

**Comment by t0mZ4** [comment]
@devaskim gotcha :)

**Comment by ashvayka** [comment]
We will add a setting for this but I will treat not as a high priority for now.

**Comment by mykhailokornieiev** [comment]
Hi @t0mZ4 
Such a feature was implemented in the v3.9 release.
You can either change the default debug value (15 minutes) in Sysadmin -> Tenant Profiles -> Debug section:

<img width="1909" height="935" alt="Image" src="https://github.com/user-attachments/assets/35c3575a-338f-4487-9b9e-44c1179c69e9" />

<img width="1031" height="360" alt="Image" src="https://github.com/user-attachments/assets/f8f72132-de57-4910-8068-89d657b59f8b" />

<img width="727" height="285" alt="Image" src="https://github.com/user-attachments/assets/5cc463bb-9dd6-482f-a3dc-15c645b8e702" />

Or in thingsboard.conf file:
`
DEBUG_SETTINGS_DEFAULT_DURATION_MINUTES`

https://thingsboard.io/docs/user-guide/install/config/#actor-system-parameters
All unconfigured tenant profiles in this case will use the default value from thingsboard.conf file 

I close this ticket as a solved one.

**Comment by t0mZ4** [comment]
yes very good!
